MapboxDirections.xcodeproj/project.pbxproj

```diff
@@ -1360,7 +1360,8 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.mapbox.MapboxDirections;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = Off;
```

Comment:
Cool, I wonder if this would've fixed issues like https://github.com/mapbox/MapboxStatic.swift/pull/66